R/Gargle-class.R

```diff
@@ -208,12 +208,14 @@ Gargle2.0 <- R6::R6Class("Gargle2.0", inherit = httr::Token2.0, list(
   },
   #' @description (Attempt to) refresh a Gargle2.0 token
   refresh = function() {
-    cred <- get("refresh_oauth2.0", asNamespace("httr"))(
-      self$endpoint, self$app,
-      self$credentials, self$params$user_params, self$params$use_basic_auth
+    cred <- refresh_oauth2.0(
```

Comment:
I was already doing something janky with this method in order to use a `Gargle2.0`-specific cache-removal procedure , so it feels fine/good to take the next step and fully implement `refresh_oauth2.0()` in gargle.